docs/code_examples/aec_rps.py

```diff
@@ -25,15 +26,16 @@
 }
 
 
-def env():
+def env(render_mode=None):
     """
     The env function often wraps the environment in wrappers by default.
     You can find full documentation for these methods
     elsewhere in the developer documentation.
     """
-    env = raw_env()
+    env = raw_env(render_mode=render_mode if render_mode != "ansi" else "human")
     # This wrapper is only for environments which print results to the terminal
-    env = wrappers.CaptureStdoutWrapper(env)
+    if render_mode == "ansi":
+        env = wrappers.CaptureStdoutWrapper(env)
```

Comment:
In line 35 there is no assignment, but just a parameter passing, thus render_mode can still be "ansi", isn't it?

Reply:
So `env.render_mode` is `human` if `render_mode` is `ansi`, otherwise `env.render_mode` is whatever `render_mode` is?

Makes sense after a couple minutes of thought, but it is a bit confusing. Gimme some minutes to review this to convince myself this is the most readable way.

Reply:
Hmm, ok. Maybe `render_mode="human" if render_mode=="ansi" else render_mode` would be more readable, along with an inline comment of something like `# switch "ansi" to "human"'`.

Reply:
Yeah basically render mode will never be ansi, because it gets converted to human otherwise

Reply:
I took it to mean more like:

```
if render_mode == "ansi":
  env.render_mode = "human"
else:
  env.render_mode = render_mode
```

But either way it's difficult to read

Reply:
Ahhh I see it now, `env.render_mode` and `render_mode` are different things. @younik Could we turn this into something a bit more readable perhaps?

Reply:
Sure, I changed it a little, let me know if this is better

Reply:
Having `internal_render_mode` seems somewhat unelegant, but fits the purpose well enough so I think that's ok.